purchase_requisition_operating_unit/__init__.py

```diff
@@ -0,0 +1,5 @@
+# © 2016 Eficent Business and IT Consulting Services S.L.
```

Comment:
It would be great if you can update the format of the copyright here to the new standard and change " Eficent Business and IT Consulting Services S.L." by "ForgeFlow S.L."

Reply:
Done, thank you @AaronHForgeFlow

Reply:
Thank you!